src/js/index.js

```diff
@@ -1,3 +1,4 @@
+import 'fetch'
 import Vue from 'vue'
```

Comment:
We've gotta rip out Vue from this project (not in this PR).  Too opinionated I think for this project.